auth/index.html

```diff
@@ -208,16 +208,23 @@ <h2>Message log</h2>
       };
 
       /**
-       * Takes a hex string (e.g. "e4567ab") and returns an array buffer (Uint8Array)
+       * Takes a hex string (e.g. "e4567abc") and returns an array buffer (Uint8Array)
        * @param {string} hexString
+       * @param {number} length: optional expected length of the resulting buffer
        * @returns {Uint8Array}
        */
-      var uint8arrayFromHexString = function(hexString) {
+      var uint8arrayFromHexString = function(hexString, length) {
```

Comment:
let's add a test to exercise this new behavior? e.g. `uint8arrayFromHexString("0001")` (should result in a buffer with a single byte) vs `uint8arrayFromHexString("0001", 2)` (should result in a buffer with 2 bytes)